### ISSUE TITLE
Cannot calculate appropriate heading level, no model id was specified 

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt-contrib-narrative#256]
### Subject of the issue
https://github.com/adaptlearning/adapt-contrib-narrative/blob/bc32cb6aebc88d4d62bc13f67da213687959986a/templates/narrative.jsx#L93

It requires an _id to calculate componentItem relative to component.
